frontend/app/src/pages/proposed-changes/details.tsx

```diff
@@ -125,10 +125,6 @@ export function Component() {
 
   const proposedChangesData = data?.[PROPOSED_CHANGES_OBJECT]?.edges?.[0]?.node;
 
-  if (!proposedChangesData) {
```

Comment:
why? if proposedchanges does not exist. We should do something (either redirect or display an error page)